src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -114,6 +130,18 @@ function IOURequestStepConfirmation({
     const isPolicyExpenseChat = useMemo(() => ReportUtils.isPolicyExpenseChat(ReportUtils.getRootParentReport(report)), [report]);
     const formHasBeenSubmitted = useRef(false);
 
+    useEffect(() => {
+        if (transaction?.participants?.findIndex((participant) => participant.accountID === transaction?.splitPayerAccountIDs?.[0]) !== -1 || iouType !== CONST.IOU.TYPE.SPLIT) {
+            return;
+        }
+
+        const payeeParticipant: Participant = {accountID: transaction?.splitPayerAccountIDs?.[0], selected: true};
```

Comment:
Why do we need this?

Reply:
Ah okay we need it as otherwise the current user won't show up here 

<img width="383" alt="Screenshot 2024-04-25 at 01 25 25" src="https://github.com/Expensify/App/assets/9680864/1c39cd46-1d3f-4f84-9e05-2056019591b9">